run

```diff
@@ -141,20 +168,33 @@ docker_django () {
 docker_yarn () {
   # Run "yarn" from the "node" image
   docker_run  \
+    ${yarn_cache_arg} \
     ${module_volumes[@]+"${module_volumes[@]}"}  `# Add any override modules as volumes`  \
-    $yarn_image $@               `# Use the image for node version 7`
+    $yarn_image $@              `# Use the image for node version 7`
 }
 
 yarn_install () {
   # Install yarn dependencies, without module overrides
   docker_run $yarn_image install
 }
 
+clean_cache () {
+  # Clean yarn cache volume
+  echo "Deleting yarn_cache Docker volume..."
+  docker volume rm yarn_cache 2>/dev/null || echo "No volume found"
+  if [ $LOCAL_YARN_INSTALLED = true ]; then
+    if [ $OPT_CACHE_CLEAN_FORCE = true ]; then
+      yarn cache clean
+    else
+      echo "#"
+      echo "# Yarn is locally installed, use --force to clean your local cache"
```

Comment:
This is making me think - maybe we should never use the local yarn cache? I think it might just confuse things. For all intents and purposes I think it makes sense to consider `docker_yarn` to be a separate application from `yarn` - it may well be a different version etc. - so maybe we should simplify things considerably by only ever storing the `docker_yarn` cache in a docker volume.